dashboard/forms.py

```diff
@@ -49,6 +49,13 @@ class Meta:
             'birthday': SelectDateWidget(years=YEAR_RANGE),
         }
 
+    def clean(self):
+        password = self.cleaned_data.get('password', None)
+        password2 = self.cleaned_data.get('password2', None)
+        if password != password2:
+            self._errors['password2'] = self.error_class(['Please make sure your passwords match'])
+        return self.cleaned_data
+
```

Comment:
This is a form for adding/creating new brothers. The password fields are to set their passwords. As to why the secretary/marshal is the one to set their passwords for them? idk

It can't really utilize User because their user doesn't exist at the point where you're filling out this form

Reply:
makes sense

iirc from when i first joined, the secretary came to my candidate meeting and had me fill out the form on their computer.  although the password never gets used lol... since we just use case login anyways